docs-ref-services/preview/cosmos-readme.md

```diff
@@ -300,26 +299,26 @@ or contact [opencode@microsoft.com][coc_contact] with any additional questions o
 
 <!-- LINKS -->
 [source_code]: https://github.com/Azure/azure-sdk-for-java/blob/azure-cosmos_4.26.0-beta.1/sdk/cosmos/azure-cosmos/src
-[cosmos_introduction]: https://docs.microsoft.com/azure/cosmos-db/
+[cosmos_introduction]: /azure/cosmos-db/
 [api_documentation]: https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/latest/index.html
-[cosmos_docs]: https://docs.microsoft.com/azure/cosmos-db/introduction
-[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[cosmos_docs]: /azure/cosmos-db/introduction
+[jdk]: /java/azure/jdk/
 [slf4j]: http://www.slf4j.org/
 [maven]: https://maven.apache.org/
 [cosmos_maven]: https://search.maven.org/artifact/com.azure/azure-cosmos
 [cosmos_maven_svg]: https://img.shields.io/maven-central/v/com.azure/azure-cosmos.svg
 [cla]: https://cla.microsoft.com
-[coc]: https://opensource.microsoft.com/codeofconduct/
```

Comment:
Is it okay to remove a different absolute link?

Reply:
Good catch, thank you!